### PR TITLE
extract wix-ui-tpa perfer to dedicated package

### PR DESCRIPTION
**Pull request by argshook**
same as https://github.com/wix/wix-style-react/pull/6933 except that it's for `wix-ui-tpa` package

---

This PR extracts `wix-ui-tpa test:perfer-bundles` script into a dedicated package.

in other words
```diff
-packages/wix-ui-tpa/.perfer/package.json
+packages/wix-ui-tpa-test-perfer/package.json
```

why?

in falcon a folder with `package.json` in it is considered package and
so falcon and yoshi tries to treat it as such.

However, in current state this was not really a package, not registered
in `lerna.json`, not in an appropriate location and used as a workaround
for `yoshi build` to have a separate `dist`.

These issues can be workaround but instead of that it's cleaner and just
easier to treat perfer test as a separate package.

Additional benefit is cleaner `scripts` section in `wix-ui-tpa/package.json`

`perfer.config.js` still remains in same place, so there's no impact on
the usage

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "test8:chromatic": "lerna run --scope wix-style-react-test-runtime test --stream",
     "test9": "lerna run --scope wix-ui-tpa test:basic --stream",
     "test10": "lerna run --scope wix-ui-tpa test:visual --stream",
-    "test11": "#",
+    "test11": "lerna run --scope wix-ui-tpa-test-perfer build --stream && lerna run --scope wix-ui-tpa-test-perfer test --stream",
```
Comment (argshook) [inline package.json:23]: wix-style-react has `test7:performance` as a separate perfer run in parallel build.
wix-ui-tpa should also have it as separate run

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6939/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6939

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6939/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6939

**Pull request merged**